list_rentals.rb

```diff
@@ -0,0 +1,40 @@
+require './app'
+require './person'
+require './student'
+require './teacher'
+require './nameable'
+require './book'
+require './rental'
+
+def list_person_rentals
+  if @persons.empty?
+    puts 'No people found'
+    puts 'Enter 3 to add people. After, proceed to create rentals for  them'
+    select_option
+  else
+    puts 'List of available people:'
+    puts ''
+    @persons.each do |person|
+      puts "#{[person.class]} Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+    end
+    puts ''
+    print 'ID of person: (Choose "ID" from the above list)'
+    puts ''
+    person_id = gets.chomp.to_i
+    puts ''
+    puts 'Rentals:'
+    @rentals.each do |rental|
+      puts rental.person.id
+      puts person_id
+      if rental.person.id.to_i == person_id
+        puts "Date: #{rental.date}, Book: #{rental.book.title} by #{rental.book.author}"
+      else
+        puts 'Sorry, no rentals found!'
+        puts ''
+        puts 'Enter 5 to create a rental for the person'
+        select_option
+      end
+    end
```

Comment:
- [ ] As option **5** for creating a rental is not working, it is impossible to also check if the **6th option of listing all rentals for a given person is** working as expected. By this I mean, once you have fixed the issue of creating a rental, you should make sure the user is able to list **all rentals for a given person** to make your App be complete. 

Good :+1: job as other options are working as expected